src/admin/components/driver/DriverTableRow.tsx

```diff
@@ -58,6 +60,13 @@ export default function DriverTableRow({ user, index }: DriverTableRowProp) {
         </span>
       </td>
       <td className="px-2 py-4">
+        <button
+          onClick={handleHistory}
+          title="History"
+          className="bg-transparent border-1  rounded-md py-1 px-2 font-normal text-blue-600 border-blue-600 hover:bg-blue-600 hover:text-white  text-base"
```

Comment:
Remove extra spacing in between class names and try to reduce number of classNames.